config/schedule.rb

```diff
@@ -25,3 +25,8 @@ def local(time)
 every 3.hours, roles: [:cron] do
   rake 'refresh_materialized_db_views'
 end
+
+every :week, at: local('02:00 am'), roles: [:cron] do
```

Comment:
Let's do this every month instead while we figure it out. If we do it every week, I feel that by the time we identify and fix issues from one seed round, the next seed round will run again.

Reply:
This will run on every environment right?? I'd rather not rely on protection in our script against running in production. Can we somehow add this task to a separate role that is only spun up on Sandbox, etc, and deliberately omitted from our production environments?